documents/subprojects.md

```diff
@@ -19,24 +19,39 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Apache Celix - Subprojects
+# Apache Celix - Libraries and Bundles
 
-Apache Celix is organized into several subprojects. The following subproject are available:
+The Apache Celix project is organized into several libraries, bundles, group of bundles and examples.
+
+## Core Libraries
+The core of Apache Celix is realized in the following libraries:
 
 * [Framework](../libs/framework) - The Apache Celix framework, an implementation of OSGi adapted to C and C++11.
-* [Utils](../libs/utils/README.md) - The Celix utils library, containing a wide range of utils functions (string, file, C hashmap, C arraylist, etc)
-* [Examples](../examples) - A Selection of examples showing how the framework can be used.
-* [Log Service](../bundles/logging/README.md) - A Log Service logging abstraction for Apache Celix.
-  * [Syslog Writer](../bundles/logging/log_writers/syslog_writer) - A syslog writer for use in combination with the Log Service.
-* [Shell](../bundles/shell/README.md) - A OSGi C and C++11 shell implementation.
-* [Pubsub](../bundles/pubsub/README.md) - An implementation for a publish-subscribe remote message communication system. 
-* [HTTP Admin](../bundles/http_admin/README.md) - An implementation for the OSGi HTTP whiteboard adapted to C and based on civetweb.
-* [Remote Services](../bundles/cxx_remote_services) - A C++17 adaption and implementation of the OSGi Remote Service Admin specification.
-* [Components Ready Check](../bundles/components_ready_check/README.md) - A bundle which checks if all components are ready.
+* [Utils](../libs/utils/README.md) - The Celix utils library, containing a wide range of general utils and 
+                                     OSGi supporting types (properties, version, filter, string utils, file utils, etc).
 
-Standalone libraries:
+## Standalone Libraries
+Apache Celix also provides several standalone libraries which can be used without the framework:
```

Comment:
It will be helpful to mention that building them with Conan involves only turning on one option.

Reply:
I will add this